cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -64,6 +64,16 @@ var _ = Describe("[sriov] [multinetworkpolicy] MultiNetworkPolicy integration",
 	sctpEnabled := false
 	sriovEnabled := false
 
+	BeforeEach(func() {
+		// Tests can be triggered by setting `FEATURE=sriov` and can fail because the
+		// feature is not enabled.
+		crdPresent, err := np.IsMultinetPolicyCRDPresentOnCluster()
+		Expect(err).ToNot(HaveOccurred())
+		if !crdPresent {
+			Skip("feature [multinetworkpolicy] not enabled on cluster.")
```

Comment:
I am not a big fan of skips. A bug in the check will result in a green CI. Wouldn't it be better to instruct the test runner to skip? Possibly with a meaningful failure that suggests to focus on sriov and skip multinetworkpolicy?

Reply:
yup, and providing a good reason for that so the user is aware.